authorizer/logger.js

```diff
@@ -0,0 +1,21 @@
+import * as winston from 'winston';
+
+const isDevelopment = process.env.NODE_ENV === 'development';
+const MAX_LEVEL = process.env.MAX_LOG_LEVEL || (isDevelopment ? 'debug' : 'info');
```

Comment:
Why `MAX_LEVEL` is in caps?

Reply:
https://google.github.io/styleguide/jsguide.html#naming-constant-names